README.md

```diff
@@ -28,5 +28,9 @@ Olympus was ideated by Zeus and built by a distributed psuedo-anonymous team.
 
 No one. Olympus is DAO-governed. All decisions are formed by community members on the forum and made by token holders through snapshot voting.
 
-
-
+## Official links
+1. [Website](https://olympusdao.finance)
+2. [Twitter](https://twitter.com/OlympusDAO)
+3. [Medium](https://olympusdao.medium.com)
+4. [Discord](https://discord.com/invite/olympusdao)
+5. [Forum](https://forum.olympusdao.finance)
```

Comment:
Would bullet points be enough or do you have a particular reason for an ordered list?

Reply:
Maybe add the following as well:
6. [Snapshot](https://vote.olympusdao.finance)

Reply:
> Maybe add the following as well:
> 6. [Snapshot] http://vote.olympusdao.finance/

Should I use scattershot link (https://scattershot.page/#/olympusdao.eth/) instead? I can only access http://vote.olympusdao.finance after disabling HTTPS-only mode, other users may find it suspicious. Also, the link would ultimately redirect to scattershot page

Reply:
+1 on TLS and Scattershot link.

Reply:
> Would bullet points be enough or do you have a particular reason for an ordered list?

Didn't think too much when adding this. I will switch to bullet points since there is no reason to order them